main.js

```diff
@@ -30,5 +30,29 @@ client.on('ready', () => {
         }
     });
 });
+client.on('message', message => { //Ten event wykonuje się, gdy bot wykryje wiadomość
+    const pingEmbed = new Discord.MessageEmbed()
+	.setColor('#eb1540')
+	.setTitle('Nie wołaj wszystkich!')
+    .setThumbnail('https://cdn.discordapp.com/attachments/617673807213232128/852887484542615572/Pingsock.png')
```

Comment:
a tego obrazka nie można umieścić w repozytorium i wysyłać? Jak nie to może gdzieś inaczej go zahostować? Bazowanie na zdjęciu wstawionym w losowej wiadomości wydaje się niezbyt dobra, nawet jeśli ten link nie wygasa to jak zniknie wiadomość, kanał czy coś, to przestanie to działać